### PR TITLE
News post about project state

### PR DESCRIPTION
**Pull request by nsg**
This is a draft post with various project update, what do you think?

### CODE REVIEW

File: _posts/2017-07-26-short-term-plans.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+> Make a voxel based game that’s focused on massive multiplayer, a simple client with most of the game logic in the server. The server is extendable with plugins.
+
+This is still true, our server is powerful and with Akka it spawns most things inside Actors that scales perfectly over multiple cores, or possible even several servers. The plugins are powerful and have control over both the world, interactions with the player and even has control over the players UI. Most of the core logic that makes the game fun, and a game, are written with the plugin system. Some of the plugins are bundled with the server, others are written in Java and are hosted as separate projects under the [Konstructs organization on GitHub](https://github.com/konstructs).
+
+When it comes to the server the thing we need to improve is easy of use, it's a little hard to understand hot to get started with a new plugin, and it can be a little tricky as a server administrator to configure and manage the plugins. To solve this we plan to:
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: hot -> how

File: _posts/2017-07-26-short-term-plans.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+* Improve documentation, the public API has excellent documentation but some interfaces need to be improved and we also need examples and guides how to get started. There will be a few quickstart guides soon!
+* Expose more internal features in the public API, most inventory logic is still part of the internal API, we hope to move it to the public API soon.
+* Make distributions, the core server is just plain boring. It's a platform for plugins, and they are what makes the game fun to playr. We plan to release a pre-packaged official distribution of the server with most of the plugins running at play.konstructs.org for easy install.
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: playr -> play

File: _posts/2017-07-26-short-term-plans.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+We played with the idea to rewrite the client in something else, like Go or Rust but decided that C++ ecosystem and all the available integrations are worth a little plain. We also looked what options of graphic engines to use out there, but at the moment that's not a focus. Future will tell!
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: plain -> pain

File: _posts/2017-07-26-short-term-plans.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+* We will drop NanoGUI from the project, we only used a small part of the project and without NanoGUI it will be easier to upgrade to a newer version of GLFW, support of Wayland is then possible.
+* We plan to implement the ability for plugins to register GUI textures and display them on the inventory UI, this will make interaction with blocks with inventories more interesting, especially machines. Everything needed to implement buttons are also there. 
+* Block inventory management will be improved, we plan to make a system that standardizes a default input and output inventory to blocks that support it. This allows other blocks to pick up or place items. This make things like transport pipes or machines that places the output in a chest possible.
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: "or place items." -> " or place items from or into other blocks."

File: _posts/2017-07-26-short-term-plans.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Snaps allows us to support Arch, Debian, Gentoo, Fedora, openSUSE, Ubuntu and plenty more from a single package. To focus on just one package format for Linux makes our work easier and we can focus more time at the actual game. If you like to maintain a 3rd-party repository or be a real maintainer for a official one you are of course welcome to do so.
+
+It is probably also a good idea to think about distribution under Windows, maybe it's time to use a installer to give the Windows users their well known wizard type of install.
+
+## Over the horizon
+
+With the above done we plan to focus on gaming experience and fun, this work will mainly be focused around plugins, the official distribution and play.konstructs.org with is the official server.
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: with -> which

File: _posts/2017-07-26-short-term-plans.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+layout: news
+title: Short term plans
+by: Stefan
+excerpt: Time for a update with our short term plans
+---
+
+# Short term plans
+
+What direction is the project heading you may wonder? Our overall [roadmap](/documentation/roadmap/) is still true but how do we plan to get there from this point? Lets discuss that by citing the roadmap:
```
Comment (petterarvidsson) [inline _posts/2017-07-26-short-term-plans.md]: "You may wonder in which direction the project is heading?"

"but" -> ", but"

**Review by petterarvidsson — CHANGES_REQUESTED** [review]
Please fix the typos, but otherwise this is a great update :+1:

**Comment by nsg** [comment]
still a draft ... so I will probably rewrite it some more. So, not to long post then?

**Comment by petterarvidsson** [comment]
I like that it is a longer one, there a re lots of things to write about.

**Pull request merged**